lib/tbot/tbot.go

```diff
@@ -170,6 +170,26 @@ func (b *Bot) Run(ctx context.Context) (err error) {
 		)
 	}
 
+	if b.cfg.DiagSocketForUpdater != "" {
```

Comment:
@boxofrad - do you foresee any issues with multiple replicas of the diagnostics service running ? From our current architecture, I don't see any problems but figured you would know with more certainty.